.projenrc.js

```diff
@@ -676,7 +678,7 @@ gitlabMain.createNestedTemplates({
       'etl-unit-test': {
         stage: 'build',
         image: {
-          name: 'public.ecr.aws/docker/library/gradle:7.6-jdk11',
+          name: 'public.ecr.aws/docker/library/gradle:7.6-jdk17',
```

Comment:
You need to run `npx projen` to re-generate the .gitlab workflow locally. Then submit the changes too.

Reply:
re-generateed